src/Template/SentencesLists/show_angular.ctp

```diff
@@ -19,6 +19,7 @@ $this->set('title_for_layout', $this->Pages->formatTitle($listName));
 ?>
 
 <div id="annexe_content">
+    <?php $this->Lists->displayFilterByLangDropdown($listId, $filterLanguage, $translationsLang); ?>
```

Comment:
How about calling `CommonModules->createFilterByLangMod()` instead?

Reply:
createFilterByLangMod seems to interfere with the `actions` dropdown, i would need to do some extra refactoring to make it work it seems. Is there a particular issue with current function? I found it used elsewhere in the codebase.

Reply:
You are welcome to do such refactoring if you want. You can as well display the "filter by lang" block as separate from the "actions" block, it's okay.

By the way, `displayFilterByLangDropdown()` is a function you introduced is this pull request. While there may not be any problem with it per se, the more we reuse existing code, the less chances we introduce new bugs and the easier it gets to maintain the code.

Reply:
I agree, I think more of the helper functions can be made to be reusable across views. 

Sorry for bad phrasing in previous comment, I meant `displayFilterByLangDropdown()` is near identical to `Lists->displayTranslationsDropdown()`. My function is basically `Lists->displayTranslationsDropdown()` with just the path building, and options array changed.

It seems `CommonModules->createFilterByLangMod()` assumes the last parameter is the `lang` this makes it incompatible with the pattern I was using `/:id/:lang/:translationsLang` which is based on `sentences/show_all_in/:lang/:translationsLang`.

Reply:
You're correct, we cannot reuse createFilterByLangMod here.